dotnet-ory-network/src/ExampleApp/Properties/launchSettings.json

```diff
@@ -0,0 +1,37 @@
+﻿{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:12416",
+      "sslPort": 44326
+    }
+  },
+  "profiles": {
+    "http": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "applicationUrl": "http://localhost:5286",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "https": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "applicationUrl": "https://localhost:7092;http://localhost:5286",

```

Comment:
Haven't seen port `7092` anywhere else 🤷

Reply:
This is generated via `dotnet new mvc` and is part of the starter template.

My idea was to keep the changes made to the template at a minimum so it can be used as a reference. Alternatively I could strip everything out, that is not relevant to the specific use case of integrating Ory Network.

Any preferences on your part @alnr @aeneasr ?